vllm/worker/model_runner.py

```diff
@@ -969,9 +971,11 @@ def vocab_size(self) -> int:
         return self.model_config.get_vocab_size()
 
 
-class CUDAGraphRunner:
+class CUDAGraphRunner(nn.Module):
```

Comment:
This got accidentally got removed in an upstream sync, right? Should we add a comment with a warning not to do that again, and explaining why?

Reply:
done!